src/Bridges/bridgeoptimizer.jl

```diff
@@ -227,32 +227,60 @@ function MOI.get(b::AbstractBridgeOptimizer,
                  attr::MOI.AbstractConstraintAttribute,
                  ci::CI)
     if is_bridged(b, typeof(ci))
-        if MOI.is_set_by_optimize(attr)
-            MOI.get(b, attr, bridge(b, ci))
-        else
+        if attr isa MOI.ConstraintName || attr isa MOI.ConstraintFunction ||
```

Comment:
For consistency with `set` and `supports`, it might be better to create a specific method for `ConstraintName` and a specific method for `ConstraintFunction`/`ConstraintSet`.